src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Coalesce.cs

```diff
@@ -22,8 +22,6 @@ internal sealed class CoalesceFunction : BuiltinFunction
     {
         public override bool IsSelfContained => true;
 
-        public override bool SupportsParamCoercion => false;
```

Comment:
>SupportsParamCoercion => false; [](http://example.com/codeflow?start=29&length=31)

Coercion is done in CheckTypes if the function takes arguments of different types